hw/hw2-baseball alternate method table 3.sql

```diff
@@ -0,0 +1,35 @@
+-- Alternate method for third table in BDA_696 HW2
+-- WORKS FOR MYSQL ONLY, WILL NOT RUN IN MARIADB
+-- Significantly faster than self-join method
+
+-- BEFORE RUNNING, disable full group by in mySQL:
+-- SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));
```

Comment:
Thank you for placing this command, this information helped some of my own issues as well